src/index.ts

```diff
@@ -1521,6 +1638,7 @@ export interface Params {
 	passthrough?: AnyObject;
 	passthroughByMethod?: { [method in ODataMethod]?: AnyObject };
 	options?: ODataOptions;
+	retryParameters?: RetryParameters;
```

Comment:
Can you name this just `retry` please
```suggestion
	retry?: RetryParameters;
```